src/components/app.jsx

```diff
@@ -111,11 +108,9 @@ const App = ({ isPublic }) => {
       <HashRouter>
         <Layout monoColumn={true}>
           {!isPublic && isMobile && (
-            <BarContextProvider webviewService={webviewService}>
-              <BarCenter>
-                <AppTitle>{appName}</AppTitle>
-              </BarCenter>
-            </BarContextProvider>
+            <BarCenter>
+              <AppTitle>{appName}</AppTitle>
+            </BarCenter>
```

Comment:
So there is no need to have this commit but also this one https://github.com/cozy/cozy-notes/pull/260/commits/54cd30c3ea07ba6a56e2ebfa0efe86af8b1f3279 

Can you remove them please?